modules/reporting/app/models/cost_query.rb

```diff
@@ -85,6 +85,25 @@ def self.exists_in?(project, user)
     public(project).or(private(project, user)).exists?
   end
 
+  # rubocop:disable Metrics/AbcSize
+  def self.build_query(project, filters, groups = {})
```

Comment:
Just a hint, you can just append the directive after the method definition. Then the cop is disabled in that method, and you don't need to manually enable it

```ruby
def self.build_query(project, filters, groups = {}) # rubocop:disable Metrics/AbcSize
```